ogr/services/github.py

```diff
@@ -102,6 +106,13 @@ def __init__(
             )
         self.read_only = read_only
 
+    def __str__(self) -> str:
+        return (
+            f"Github-Project("
+            f"namespace: '{self.namespace}'"
+            f"repo: '{self.repo}')"
+        )
```

Comment:
Personally, I like the instantiation-like syntax. You can sometimes use it to "recreate" the original object:

```python
>>> str(GithubProject(repo="my_repo", namespace="my_namespace"))
'GithubProject(repo="my_repo", namespace="my_namespace")'
```

Reply:
Yeah, I will make required changes.
Thanks for your feedback.